doc/U-19_Initialize_game.md

```diff
@@ -0,0 +1,22 @@
+# U-19: Initialize game
+**Actor:** Admin
+
+**Preconditions:** -
+
+**Postconditions:**
+
+ - Team status for every cypher is reset (first cypher is in status pending)
+ - Every team has no taken hint
+ - Teams have 0 points
+
+## Main Scenario
+
+ 1. **Admin:** decides to start the game
+ 2. **System:** asks user for password
```

Comment:
ask "admin", je to mysleno jako use case pro admina. Hru zahajuje admin vyresetovanim DB pro vsechny týmy nebo nebo kazdy tym pro sebe prvnim prihlasenim ?

Reply:
Přesně tak. Může to udělat admin na začátku hry pro všechny týmy.
Admin přidá týmy, podle toho kdo přijde na šifrovačku a pak zahájí hru.

@mjanousek A když se tak Jíťa ptá, to asi vyloučí týmy, které přijdou o něco pozdě a chtěly by se přidat do hry.

Reply:
Dobry point. Tim padem bude teda idealni to, ze si kazdy ucastnik spusti hru sam ;)